db_management.py

```diff
@@ -1,31 +1,52 @@
-import sqlite3
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy.ext.declarative import declarative_base
 
 
-def new_user_creation(username_text, pass_text):
-    conn = sqlite3.connect('data_base.db')
-    curs = conn.cursor()
-    params = (username_text, pass_text)
-    curs.execute("INSERT INTO users VALUES (NULL, ?, ?)", params)
-    conn.commit()
-    conn.close()
+engine = create_engine('sqlite:///data_base.db')
+conn = engine.connect()
+Session = sessionmaker(bind=engine)
+session = Session()
+Base = declarative_base()
+
+
+class User(Base):
+    __tablename__ = 'users'
+    id = Column(Integer, primary_key=True)
+    Username = Column(String, nullable=False, unique=True)
+    Password = Column(String, nullable=False, unique=True)
+
+
+def create_connection():
```

Comment:
You don't create a connection here, you create all your tables. Let's rename the method function to something like `def create_db_tables():`? wdyt?

Reply:
sure